SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -140,12 +142,21 @@ TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> SpatialLoadBalanceE
 		const SpatialGDK::AuthorityIntent* AuthorityIntentComponent = StaticComponentView->GetComponentData<SpatialGDK::AuthorityIntent>(EntityId);
 		if (AuthorityIntentComponent == nullptr)
 		{
-			// This happens if the authority intent component is removed in the same tick as a request is queued, but the request was not removed from the queue- shouldn't happen.
+			// This happens if the authority intent component is removed in the same tick as a request is queued, but the request was not removed from the queue - shouldn't happen.
```

Comment:
This is another reason we should just delete the queue before making more changes to the enforcer. It introduces a lot of complexity for no benefit. This PR is already up and I'm happy to approve once we are sure it's correct, but I would support removing the queue first and making this PR simpler as a result.